canonical-websites/www.canonical.com
# Several styling tweaks

Dependant on PR: https://github.com/ubuntudesign/canonical-vanilla-theme/pull/15 (now merged)

Done:
- fix mobile styling for navigation
- remove any unneeded CSS
- made small overrides for minor bugs (hiding bullets, etc.)

QA:
Make sure that your node_modules version of canonical-vanilla-theme and, inside that, vanilla-framework are on the bleeding-edge from git.

Check the navigation works on mobile viewports.
Check on /careers that the list linked by lines don't have bullets.

Note: there still may be some minor styling inconsistencies between this and live but there should be no _new_ issues from these changes.
